cmd/amp-cluster/cli_node_count.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"github.com/appcelerator/amp/cmd/cluster-server/servergrpc"
+	"github.com/spf13/cobra"
+	"os"
+	"time"
+)
+
+// NodeCountCmd is the node couunt command
+var NodeCountCmd = &cobra.Command{
+	Use:   "count",
+	Short: "count containers/images in nodes",
+	Long:  `count containers/images in nodes`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return client.containerCount(cmd, args)
+	},
+}
+
+func init() {
+	NodeCmd.AddCommand(NodeCountCmd)
+	NodeCountCmd.Flags().BoolP("follow", "f", false, "follow node list")
+	NodeCountCmd.Flags().StringP("node", "n", "", "specify the node onto apply the purge, default all")
+}
+
+func (g *clusterClient) containerCount(cmd *cobra.Command, args []string) error {
+	req := &servergrpc.GetNodesInfoRequest{}
+	follow := false
+	req.Node = cmd.Flag("node").Value.String()
+	if cmd.Flag("follow").Value.String() == "true" {
+		follow = true
+	}
+	if follow {
+		fmt.Println("\033[2J\033[0;0H")
```

Comment:
These printlns are both arcane and used a number of places maybe a `g.followStart(follow bool)` and `g.followLn(follow bool)`